### PR TITLE
MGMT-4137 - Add day2 usage

### PR DESCRIPTION
**Pull request by slaviered**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -1368,13 +1374,17 @@ func (b *bareMetalInventory) InstallHosts(ctx context.Context, params installer.
 		}
 	}
 
+	//report day2 feature usage and save the result within the same transaction
+	b.setUsage(true, usage.Day2HostUsage, &map[string]interface{}{
+		"day2_host_count": len(cluster.Hosts)}, usages)
```
Comment (rollandf) [inline internal/bminventory/inventory.go:1379]: Not all `cluster.Hosts` are always installed. Only the one in `Known` state

**Review by rwsu — COMMENTED** [review]
Please add a git commit message.

**Comment by flaper87** [comment]
@slaviered are you planning to rebase/work on this PR?

If not, do you think we can close it?

**Pull request closed without merging**